src/components/react_canvas/reactive_chart.tsx

```diff
@@ -349,8 +349,16 @@ class Chart extends React.Component<ReactiveChartProps, ReactiveChartState> {
       chartRotation,
       chartTransform,
       setCursorPosition,
+      isChartEmpty,
     } = this.props.chartStore!;
 
+    if (isChartEmpty) {
+      return (
+        <div className="echUnavailable">
```

Comment:
You'll want to be more specific with your class naming so that you don't accidentally create too generic of a class that it's not easily recognizable as to where the class is applied and/or that it doesn't get overridden. So for this component, you'd likely start with the exported class name and append `unavailable` as a child style like:

```suggestion
        <div className="echReactiveChart__unavailable">
```